fix(ivy): not applying camelCased style properties

crisbeto opened the pull request:
Fixes Ivy not applying properties that are set in camelCase, because it goes through the `CSSStyleDeclaration` API via `setProperty` and `removeProperty` which requires for [the values to be in dash-case](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty).

**Note:** I opted to let the browser normalize the value, rather than convert it to dash-case during compile time, because there are some special cases like browser-prefixed properties where we might not normalize it in-line with the browser.

This PR fixes FW-579.

kara commented on packages/core/test/render3/styling/class_and_style_bindings_spec.ts:
```diff
@@ -380,6 +380,41 @@ describe('style and class based bindings', () => {
+            type: Comp,
+            selectors: [['comp']],
+            factory: () => new Comp(),
+            consts: 2,
```
```suggestion
            consts: 1,
```

kara reviewed (approved):
LGTM, one nit

crisbeto committed:
fix(ivy): not applying camelCased style properties

Fixes Ivy not applying properties that are set in camelCase, because it goes through the `CSSStyleDeclaration` API via `setProperty` and `removeProperty` which requires for [the values to be in dash-case](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty).

**Note:** I opted to let the browser normalize the value, rather than convert it to dash-case during compile time, because there are some special cases like browser-prefixed properties where we might not normalize it in-line with the browser.

This PR fixes FW-579.
